cmd/minikube/cmd/get_openshift_versions.go

```diff
@@ -26,8 +26,9 @@ import (
 // getVersionsCmd represents the ip command
 var getVersionsCmd = &cobra.Command{
 	Use:   "get-openshift-versions",
-	Short: "Gets the list of available OpenShift versions available for minishift.",
-	Long:  `Gets the list of available OpenShift versions available for minishift.`,
+	Short: "Gets the list of OpenShift versions available for Minishift.",
+	Long:  `Gets the list of OpenShift versions available for Minishift.`,
+	//MOREINFO: Do we want to elaborate on usage or examples? Like is this STDOUT only, can we output to a file, do we want to pass this onto something else?
```

Comment:
The NEEDINFO comments were basically me asking for clarification or extra info about the content I was editing, with the hope of resolving them in the scope of this PR. Of course if we need to create related issues (for example, if the answer to my question is too big for the scope of the PR) then sure let's go ahead and do that, but I'd like to keep them in the code until resolved if it doesn't get in the way.